bindings/native/src/wrapper.rs

```diff
@@ -181,13 +179,13 @@ impl StrongholdWrapper {
             vault_path: VAULT_PATH.as_bytes().to_vec(),
         };
 
-        let chain = Chain::from_u32_hardened(vec![
```

Comment:
Just checking, was this previously hardening? Or just expecting already hardened values?

Reply:
Thanks, my bad, this is bug. Just lost chain hardening for Ed25519.

Reply:
Fixed